### PR TITLE
Tri 274 implementing aspect models into irs

### PR DESCRIPTION
**Pull request by mkanal**
new api model v0.2 is available now.

### CODE REVIEW

**Comment by mkanal** [comment]
@ds-jkreutzfeld  Pls review the pull request. Thx

File: uml-diagramms/partRelationshipsWithInfos-modelv0.2.puml [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+    asyncFetchedItems : AsyncFetchedItems
+}
+
+entity AsyncFetchedItems {
+    ---
+    * queued : Number (Default :0)
+    * running : Number (Default :0)
+    * complete : Number (Default :0)
+    * failed : Number (Default :0)
+    * lost : Number (Default :0)
```
[the remark this answers is not in the document]
Comment (mkanal) [reply uml-diagramms/partRelationshipsWithInfos-modelv0.2.puml:36]: @jzbmw Could I forward this question to you. 
IMHO 
failed: Endpoint is not responding. HTTP 4XX 
lost: Endpoint not available? -> HTTP 5XX
Do we want to have a difference between these return codes?

[the remark this answers is not in the document]
Comment (jzbmw) [reply uml-diagramms/partRelationshipsWithInfos-modelv0.2.puml:36]: For Release 1 the "lost"  can be removed

[the remark this answers is not in the document]
Comment (mkanal) [reply uml-diagramms/partRelationshipsWithInfos-modelv0.2.puml]: @ds-jkreutzfeld  lost was removed.

**Pull request closed without merging**